ironic-config/apache2-ironic-api.conf.j2

```diff
@@ -10,7 +10,7 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-Listen {{ env.IRONIC_LISTEN_PORT }}
+Listen {{ env.IRONIC_URL_HOST }}:{{ env.IRONIC_LISTEN_PORT }}
```

Comment:
I think you need to respect LISTEN_ALL_INTERFACES here

Reply:
Thanks, that's a good point. I was under the impression that it was not important as the previous setting didn't care about that. Making change